web-console/src/views/tasks-view.tsx

```diff
@@ -521,9 +521,10 @@ ORDER BY "rank" DESC, "created_time" DESC`);
               const previewCount = countBy(previewValues);
               return <span>{Object.keys(previewCount).sort().map(v => `${v} (${previewCount[v]})`).join(', ')}</span>;
             },
-            sortMethod: (status1: string, status2: string) => {
+            sortMethod: (d1, d2, desc) => {
               const statusRanking: any = this.statusRanking;
-              return statusRanking[status1] - statusRanking[status2];
+              const timeDifference = Date.parse(d1.created_time) - Date.parse(d2.created_time);
+              return statusRanking[d1.status] - statusRanking[d2.status] || (desc ? timeDifference : -timeDifference);
```

Comment:
Why is the `desc` parameter only considered in the tie breaker?

Reply:
This is to keep row with later `created_time` always on top no matter it's desc or asc. We can use the default order for status.

Reply:
I think that is not what we want. I think the sort should behave honestly meaning that buy default Running is on top and the latest created time is on top, but if you reverse the order then the entire list reverses.

Reply:
Will change the code and the descriptions